apps/client/electron/windows/sign_windows_executable.action.mjs

```diff
@@ -19,7 +19,7 @@ import {dirname, resolve} from 'path';
 import {fileURLToPath, pathToFileURL} from 'url';
```

Comment:
I think we have something equivalent for the manager. We'll probably want to put this under a shared infrastructure windows signing folder at some point.